.gitignore

```diff
@@ -1,8 +1,13 @@
 _local
-node_modules
 .phpunit.result.cache
 
 # Local Netlify folder
 .netlify
 
+# Node / Deno
+node_modules
+.npmrc
```

Comment:
Why to ignore this one?

Reply:
Hmm, so the `Gitignore` changes came from using `npx gitignore`

I see that we already have a `.npmrc` committed, and we're not really using `npm` anywhere. I don't think `pnpm` reads the `.npmrc` file.

So I've removed it from the branch

Reply:
https://pnpm.io/8.x/npmrc 😉